### PR TITLE
Small changes to unblock test scenario

### PR DESCRIPTION
**Pull request by chlowell**
cc @twsouthwick

### CODE REVIEW

File: src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectPropertyDescriptor.cs [hunk 1]
```diff
@@ -975,7 +975,9 @@ protected override void FillAttributes(IList attributes)
         /// </devdoc>
         public override object GetValue(object component)
         {
-            Debug.WriteLine($"[{Name}]: GetValue({component?.ToString() ?? "(null)"})");
+            // TODO determine whether this is valuable. If so, find another way to do it.
+            // component's ToString method can call this method and provoke a stack overflow.
+            //Debug.WriteLine($"[{Name}]: GetValue({component?.ToString() ?? "(null)"})");
```
Comment (twsouthwick) [inline src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectPropertyDescriptor.cs]: Maybe replace it with a component?.GetType().Name call

Comment (chlowell) [inline src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectPropertyDescriptor.cs]: Sure, that's a good substitute.

File: src/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeConverter.cs [hunk 2]
```diff
@@ -85,6 +85,14 @@ public object ConvertFromString(string text)
         /// <devdoc>
         ///    <para>Converts the specified text into an object.</para>
         /// </devdoc>
+        public object ConvertFromString(ITypeDescriptorContext context, string text)
```
Comment (twsouthwick) [inline src/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeConverter.cs]: Is this in the contract? Should we add it?

**Comment by twsouthwick** [comment]
LGTM

File: src/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeConverter.cs (hunk 2 above)
Comment (chlowell) [inline src/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeConverter.cs]: It is.

**Pull request merged**